components/namespace-claim-cleaner/base/namespace-claim-configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: namespace-claim-configmap
+  namespace: namespace-claim-cleaner
+data:
+  namespace-claim-cleaner.sh: |
+    #!/bin/bash
+
+    # Get namespaces with label namespaces.eaas.konflux-ci-dev
+    LABEL="namespaces.eaas.konflux-ci-dev"
```

Comment:
We can't list the namespaces generated by the claim. They're not available on the cluster where this code runs.

This is more complex that it needs to be. We want to list and delete the stale `namespaces.eaas.konflux-ci-dev` resources.